cMake/FreeCAD_Helpers/PrintFinalReport.cmake

```diff
@@ -1,182 +1,128 @@
+    simple(QtXml ${QtXml_VERSION})
     if (BUILD_GUI)
-        report(QtWidgets ${QtWidgets_VERSION})
-        report(QtPrintSupport ${QtPrintSupport_VERSION})
-        report(QtOpenGL ${QtOpenGL_VERSION})
-        report(QtSvg ${QtSvg_VERSION})
-        report(QtUiTools ${QtUiTools_VERSION})
-        report(QtConcurrent ${QtConcurrent_VERSION})
-        if(BUILD_WEB)
-            report(QtWebEngineWidgets ${QtWebEngineWidgets_VERSION})
-        else()
-            report(QtWebEngineWidgets "not needed (BUILD_WEB is OFF)")
-        endif()
-        if(BUILD_DESIGNER_PLUGIN)
-            report(DesignerPlugin "${DESIGNER_PLUGIN_LOCATION}/${libFreeCAD_widgets}")
-        else()
-            report(DesignerPlugin "not built (BUILD_DESIGNER_PLUGIN is OFF)")
-        endif()
+        conditional(QtWebEngineWidgets BUILD_WEB "not needed (BUILD_WEB is OFF)" ${QtWebEngineWidgets_VERSION})
+        conditional(DesignerPlugin BUILD_DESIGNER_PLUGIN
+                    "not built (BUILD_DESIGNER_PLUGIN is OFF)"
+                    "[${DESIGNER_PLUGIN_LOCATION}/${libFreeCAD_widgets}]"
+        )
     else()
-        report(QtWidgets "not needed")
-        report(QtPrintSupport "not needed")
-        report(QtOpenGL "not needed")
-        report(QtSvg "not needed")
-        report(QtUiTools "not needed")
-        report(QtConcurrent "not needed")
-        report(QtWebKitWidgets "not needed")
-    endif()
-
-    if(DEFINED MACPORTS_PREFIX)
-        if(Shiboken${SHIBOKEN_MAJOR_VERSION}_FOUND)
-            report(Shiboken "${Shiboken_VERSION} [${SHIBOKEN_INCLUDE_DIR}]")
-        else()
-            report(Shiboken "not found (only searched if MACPORTS_PREFIX is defined)")
-        endif()
-        if(PySide_FOUND)
-            report(PySide "${PySide_VERSION} [${PYSIDE_INCLUDE_DIR}]")
-            if(NOT PYSIDE_INCLUDE_DIR)
-                report(IncludeDir "Unable to find, python version mismatch?")
-            endif()
-        else()
-            report(PySide: "not found (only searched if MACPORTS_PREFIX is defined)")
-        endif()
-    endif()
-
-    if(PySide${SHIBOKEN_MAJOR_VERSION}_FOUND)
-        report(PySide "${PYSIDE_VERSION} [${PYSIDE_INCLUDE_DIR}]")
-        if(NOT PYSIDE_INCLUDE_DIR)
-            report(PySideIncludeDir "Unable to find, python version mismatch?")
-        endif()
-    else()
-        report(PySide "not found")
-    endif()
-    if(PYSIDE_TOOLS_FOUND)
-        report(PySideTools "[${PYSIDE_UIC_EXECUTABLE}] [${PYSIDE_RCC_EXECUTABLE}]")
-    else()
-        report(PySideTools "not found")
-    endif()
-
+        simple(QtWebKitWidgets "not needed")
+    endif()
+    conditional(Shiboken SHIBOKEN_FOUND "not found" "${SHIBOKEN_VERSION} Base: [${SHIBOKEN_BASEDIR}]")
+    conditional(PySide PYSIDE_FOUND "not found" "${PYSIDE_VERSION} [${PYSIDE_INCLUDE_DIR}]")
+    conditional(PySideTools PYSIDE_TOOLS_FOUND
+                "not found"
+                "uic: [${PYSIDE_UIC_EXECUTABLE}] rcc: [${PYSIDE_RCC_EXECUTABLE}]"
```

Comment:
This line exposes a problem in the code that's finding PySideTools. On Windows, the backslash character is retained in these two executable paths: until now that was harmless (and IMO actually sort of useful). However, that is breaking your macro above, because it gets interpreted as an escape character. All other Windows paths are being  modified to use a (forward) slash as a separator, which I believe is the standard way of doing it in cMake. In order to merge the above change, that needs to be addressed first.

Reply:
I'd appreciate a double-check on #8293, which will correct this.

Reply:
Go for it! I commented in your pr.